src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -6,12 +6,14 @@
 public class Messages {
     public static final String MESSAGE_DUPLICATE_PERSON = "A person with this name already exists. "
             + "Please do add tags to other persons with the same name to differentiate between them!";
+    public static final String MESSAGE_DUPLICATE_MEETING = "A meeting with this agenda already exists. ";
     public static final String MESSAGE_DUPLICATE_DETAILS = "A person with these details already exists. "
             + "Please do add tags that differentiate between them!";
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_VIEW = "This command is invalid here!";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_MEETING_DISPLAYED_INDEX = "The meeting index provided is invalid";
```

Comment:
```suggestion
    public static final String MESSAGE_INVALID_PERSON_INDEX = "The person index provided is invalid";
    public static final String MESSAGE_INVALID_MEETING_INDEX = "The meeting index provided is invalid";
```

Would these be better names for the messages?